test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -8334,21 +8334,15 @@ def cleanup_jobs(self, extend=None, runas=None):
             select_xt = 'x'
         job_ids = self.select(extend=select_xt)
 
-        # Make sure the current user is a manager. Some tests might have
-        # unset the mangers attribute. Ignore 'Duplicate entry in the list'
-        # error if the current user was already a manager
-        current_user = pwd.getpwuid(os.getuid())[0]
-        a = {ATTR_managers: (INCR, current_user + '@*')}
-        try:
-            self.manager(MGR_CMD_SET, SERVER, a, sudo=True)
-        except PbsManagerError:
-            pass
-
         # Turn off scheduling so jobs don't start when trying to
         # delete. Restore the orignial scheduling state
         # once jobs are deleted.
         sched_state = self.status(SERVER, {'scheduling'})[0]['scheduling']
-        self.manager(MGR_CMD_SET, SERVER, {'scheduling': 'False'})
+
+        # runas is required here because some tests remove current user
+        # from managers list
+        self.manager(MGR_CMD_SET, SERVER, {'scheduling': 'False'},
```

Comment:
Hi @latha-subramanian , In case multiple schedulers are present, we should be setting false for all the schedulers instead on server which will only set False on default scheduler.

Reply:
@visheshh , I have made changes such that it works for multischeds too. Please take a look

Reply:
@latha-subramanian Thanks for making changes.